docs/brand/copywriting/naming.html

```diff
@@ -70,55 +66,31 @@
+    {% header "h4", "Examples" %}
 
     <ul class="stacks-list">
-        <li>Do follow the <a href="/content/guidelines/grammar-and-mechanics/#use-sentence-casing">capitalization rules</a> for headers, sub-headers, and menu tabs.</li>
-        <li>Use sentence case for all headings (Sentence case involves capitalizing the first word, first word of a subtitle, and all proper nouns.)</li>
-        <li>Capitalize the first word of a heading</li>
-        <li>Capitalize proper or trademarked nouns (names of products, people, or specific companies)
-        <li>Lowercase for everything else</li>
+        <li>“Unlike simply sharing information, <strong>knowledge reuse</strong> takes collaboration a step further by creating a system where ideas are shared, quickly accessed, improved upon, and then put back into the system.”</li>
+        <li>“We have calculated your <strong>Knowledge Reuse</strong> metric as…”</li>
     </ul>
 
-    <table class="s-table s-table__lg fs-body2 va-top w100 wmx6 mb48">
-        <thead>
-            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1 s-table--cell6">{% icon "Clear", "mtn1 mbn1" %} Don’t</th>
-            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1 s-table--cell6">{% icon "Checkmark", "mtn1 mbn1" %} Do</th>
-        </thead>
-        <tbody>
-            <tr>
-                <td>User Roles</td>
-                <td>User roles</td>
-            </tr>
-        </tbody>
-    </table>
+    {% header "h3", "Communities & Community" %}
 
-   {% header "h4", "Product and feature names" %} 
+    <p class="stacks-copy">“Communities” and “Community” are capitalized when referring to the Stack Overflow for Teams feature.</p>
 
-    <p class="stacks-copy">
-        If a product or feature isn’t unique to Stack Overflow, <strong>don’t</strong> capitalize it (blogs, pages, etc.) If it is unique to Stack Overflow as its own product, <strong>do</strong> capitalize it. 
-    </p>
+    {% header "h4", "Examples" %}
 
-    <table class="s-table s-table__lg fs-body2 va-top w100 wmx6 mb48">
-        <thead>
-            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1 s-table--cell6">{% icon "Clear", "mtn1 mbn1" %} Don’t</th>
-            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1 s-table--cell6">{% icon "Checkmark", "mtn1 mbn1" %} Do</th>
-        </thead>
-        <tbody>
-            <tr>
-                <td>As soon as it’s ready, put your <strong class="fc-red-500">Blog Post</strong> up</td>
-                <td>As soon as it’s ready, put your <strong class="fc-green-500">blog post</strong> up</td>
-            </tr>
-        </tbody>
-    </table>
+    <ul class="stacks-list">
+        <li>With <strong>Communities</strong> you can help your teammates know what's trending, who's sharing knowledge, and where there's opportunity to contribute or learn around specific areas of interest.</li>
```

Comment:
Band name: Apostrophe consistency

```suggestion
        <li>With <strong>Communities</strong> you can help your teammates know what’s trending, who’s sharing knowledge, and where there’s opportunity to contribute or learn around specific areas of interest.</li>
```